src/app/loadout/loadout-edit/LoadoutEditSubclass.tsx

```diff
@@ -101,6 +101,7 @@ export default function LoadoutEditSubclass({
                   // plugs in the loadout and they may be different to the popup
                   onClick={plugs.length ? undefined : onClick}
                   item={subclass.item}
+                  hideSelectedSuper
```

Comment:
Maybe we just remove this prop altogether from inventory item and do this by default?